package.json

```diff
@@ -1,35 +1,34 @@
 {
-    "name": "@fluid-example/hello-world",
-    "version": "0.1.0",
-    "description": "Minimal Fluid Container & Data Object sample to implement a collaborative dice roller as a standalone app.",
-    "repository": "microsoft/FluidHelloWorld",
-    "license": "MIT",
-    "author": "Microsoft",
-    "scripts": {
-        "build": "webpack --env prod --env clean",
-        "build:dev": "webpack --env clean",
-        "start": "start-server-and-test start:server 7070 start:client",
-        "start:client": "webpack serve",
-        "start:server": "npx tinylicious@latest"
-    },
-    "dependencies": {
-        "@fluidframework/tinylicious-client": "^1.2.5",
-        "fluid-framework": "^1.2.5"
-    },
-    "devDependencies": {
-        "clean-webpack-plugin": "^3.0.0",
-        "concurrently": "^5.3.0",
-        "html-webpack-plugin": "^4.3.0",
-        "start-server-and-test": "^1.11.6",
-        "webpack": "^4.46.0",
-        "webpack-cli": "^4.8.0",
-        "webpack-dev-server": "^3.11.0"
-    },
-    "prettier": {
-        "trailingComma": "es5",
-        "tabWidth": 4,
-        "semi": true,
-        "singleQuote": false,
-        "printWidth": 100
-    }
+  "name": "@fluid-example/hello-world",
+  "version": "0.1.0",
+  "description": "Minimal Fluid Container & Data Object sample to implement a collaborative dice roller as a standalone app.",
+  "repository": "microsoft/FluidHelloWorld",
+  "license": "MIT",
+  "author": "Microsoft",
+  "scripts": {
+    "build": "webpack --env prod --env clean",
+    "build:dev": "webpack --env clean",
+    "start": "start-server-and-test start:server 7070 start:client",
+    "start:client": "webpack serve",
+    "start:server": "npx tinylicious@latest",
+    "format": "npm run prettier:fix",
+    "lint": "npm run prettier",
+    "lint:fix": "npm run prettier:fix",
+    "prettier": "prettier --check . --ignore-path ./.prettierignore",
```

Comment:
I think we can omit the --ignore-path arg here since we don't actually have an ignore file. Prettier's default policy is to use the .gitignore, so we should be okay with that.

Reply:
I think `.prettierignore` is required as lock files cannot get included in `.gitignore`